parsers/US_CA.py

```diff
@@ -194,10 +200,8 @@ def fetch_exchange(
     daily_data = []
     for i in range(0, latest_index + 1):
         h, m = map(int, csv["Time"][i].split(":"))
-        date = (
-            arrow.utcnow()
-            .to("US/Pacific")
-            .replace(hour=h, minute=m, second=0, microsecond=0)
+        date = datetime.now(tz=TIMEZONE).replace(
+            hour=h, minute=m, second=0, microsecond=0
```

Comment:
Since date is a datetime object now we need to remove the `.datetime` call on line 208 just below this part.